src/styles/default/pretty.jl

```diff
@@ -283,27 +283,26 @@ end
 
 function format_docstring(style::AbstractStyle, state::State, text::AbstractString)
     state_indent = state.indent
-    boundaries = findall(text) do character
-        character != '"'
-    end
+    start_boundary = findfirst(!=('"'), text)
+    end_boundary = findlast(!=('"'), text)
```

Comment:
Just a minor optimization while working on this part of the code (I'll revert if desired): Only the indices of the first and last non-`"` character are needed.